### ISSUE TITLE
[BUG] memory leak `v2.4.2`

### ISSUE THREAD

**Issue by bert2002** [classic-terra/core#460]
**Describe the bug**
Since a couple of hours my terra classic node is consuming all memory and in the end will get killed by the kernel (32GB within 1-2 minutes). 

There are some huge IBC transfers that seems to be happening around the same time 
[terra.txt](https://github.com/classic-terra/core/files/14986332/terra.txt)

```
Apr 16 02:43:40 terra-classic-1 terrad[2166]: 2:43AM INF IBC fungible token transfer amount=1 module=x/ibc-transfer receiver="5e3db9589dd3324ad3b060d23a556e055df6cc8f342a5a503831c14401aeb7245a3af78ef164de1464e2bf195c4dfa728cc999e3db6949c29e153847bf23918e25d221da82159200b7ed2aa0676d6093bbb5a5342cdde8a539f9dafce65104a7455b6bea93e6a463ed241b38689b6ac617b610fd773b44a9e8fe06c2cf78d70f792ef76a8c5c054febff020c8330d012047ac76b12c7363de175779bb368fdbaf434db3e74da7c606d55770135a4ec56308f71894d04f3a97a13381555e9bb18c1c1679826f4b8ad1238244684652ba7fb0f71b63de4fd9eb415c366b02c3728d3530723a0ec5f0281ce43472fa47bc387866db095c644bbd1cca9b8c038719b09c7a713eeaa7bba948e9593a11ba50cacee7ebd5808674b612b1cc3cc898d9056d7be329a353ee33f1649e53d6625baecaa3a92673a5758ac7a05b6....
````

Anyone experience the same?

Thanks,
bert

**Comment by bert2002** [comment]
Some more logs:
```
Apr 16 03:16:13 terra-classic-1 terrad[663]: 3:16AM INF failed attempting to add vote err="expected 17620361/1/2, but got 17620361/0/2: unexpected step" module=consensus
Apr 16 03:16:13 terra-classic-1 terrad[663]: 3:16AM ERR failed to process message err="error adding vote" height=17620362 module=consensus msg_type=*consensus.VoteMessage peer=c3a3156ae72d7db468758030d8c156216b9205a5 round=0
Apr 16 03:16:13 terra-classic-1 terrad[663]: 3:16AM INF failed attempting to add vote err="expected 17620361/1/2, but got 17620361/0/2: unexpected step" module=consensus
Apr 16 03:16:13 terra-classic-1 terrad[663]: 3:16AM ERR failed to process message err="error adding vote" height=17620362 module=consensus msg_type=*consensus.VoteMessage peer=c3a3156ae72d7db468758030d8c156216b9205a5 round=0
Apr 16 03:16:13 terra-classic-1 terrad[663]: 3:16AM INF failed attempting to add vote err="expected 17620361/1/2, but got 17620361/0/2: unexpected step" module=consensus
Apr 16 03:16:13 terra-classic-1 terrad[663]: 3:16AM ERR failed to process message err="error adding vote" height=17620362 module=consensus msg_type=*consensus.VoteMessage peer=c3a3156ae72d7db468758030d8c156216b9205a5 round=0
```

**Comment by bert2002** [comment]
Seems its not a bug, but huge memo's.

**Comment by nghuyenthevinh2000** [comment]
should be fixed with #463
